backend/apps/sandbox/views.py

```
@@ -235,11 +310,102 @@ def get_queryset(self):
+        payload_raw = request.query_params.get('payload', '{}')
+        
+        if not code:
+            return Response(
+                {'error': 'Code is required'},
+                status=status.HTTP_400_BAD_REQUEST
+            )
+        
+        try:
+            import json
+            payload = json.loads(payload_raw)
+        except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)